spec/model/post_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] Please I think any custom methods in the models should have a corresponding unit test so I suggest you write a unit test for  the custom method `return_5_recent_comments` :pray: